src/demo.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
+import "@fontsource/figtree";
```

Comment:
This import should (is?) be unnecessary

Reply:
At the moment, it is not working without the import

Reply:
Should we keep this import in this file for now?

Reply:
It's now removed, right? Demo works fine for me

Reply:
Yes, it works for me all across Safari, Chrome, Firefox

Reply:
Okay, then I will merge the PR